packages/mongodb-constants/src/bson-types.ts

```diff
@@ -122,6 +122,46 @@ const BSON_TYPES = [
     description: 'BSON Regex type',
     snippet: "RegExp('${1:source}', '${2:opts}')",
   },
+  {
+    name: 'LegacyJavaUUID',
+    value: 'LegacyJavaUUID',
+    label: 'LegacyJavaUUID',
+    score: 1,
+    meta: 'bson-legacy-uuid',
+    version: '0.0.0',
+    description: 'BSON Binary subtype 3 (Java legacy UUID)',
+    snippet: "LegacyJavaUUID('${1:uuid}')",
+  },
+  {
+    name: 'LegacyCSharpUUID',
+    value: 'LegacyCSharpUUID',
+    label: 'LegacyCSharpUUID',
+    score: 1,
+    meta: 'bson-legacy-uuid',
+    version: '0.0.0',
+    description: 'BSON Binary subtype 3 (CSharp legacy UUID)',
+    snippet: "LegacyCSharpUUID('${1:uuid}')",
+  },
+  {
+    name: 'LegacyPythonUUID',
+    value: 'LegacyPythonUUID',
+    label: 'LegacyPythonUUID',
+    score: 1,
+    meta: 'bson-legacy-uuid',
+    version: '0.0.0',
+    description: 'BSON Binary subtype 3 (Python legacy UUID)',
+    snippet: "LegacyPythonUUID('${1:uuid}')",
+  },
+  {
+    name: 'UUID',
```

Comment:
Previously UUID didn't show up in Compass' autocomplete. I'm thinking it was just missing. This will make it start showing up:

<img width="605" height="85" alt="Image" src="https://github.com/user-attachments/assets/06ff3d00-3075-449d-ab14-3e4514287f4d" />